src/gui/rss/rssparser.cpp

```diff
@@ -266,6 +272,11 @@ void RssParser::parseRssArticle(QXmlStreamReader& xml, const QString& feedUrl)
         else
           article["news_link"] = link;
       }
+      else if(xml.name() == "magnetURI") {
```

Comment:
Here's an example; it's a feed generated from a search term.

http://kat.cr/usearch/ubuntu%20desktop/?rss=1

The feed uses the http://xmlns.ezrss.it/0.1/ torrent namespace.

The reference for the namespace is offline, I'm guessing due to server/url moves, but I've included webarchive links to the definition files.

http://web.archive.org/web/20141204192139/http://xmlns.ezrss.it/0.1/
http://web.archive.org/web/20141029030907/http://xmlns.ezrss.it/0.1/dtd/